packages/protocol/scripts/foundry/create_and_migrate_anvil_l2_devchain.sh

```diff
@@ -45,34 +28,34 @@ CELO_DISTRIBUTION_SCHEDULE_ADDRESS=$(
 )
 
 # Set the balance of the CeloDistributionSchedule (like the Celo client would do during L2 genesis)
-# This cannot be done during the migrations, because CeloDistributionSchedule.sol does not
-# implement the receive function nor does it allow ERC20 transfers.
-# This is the only way I managed to give the CeloDistributionSchedule a balance.
+# Note: This can't be done from the migration script, because CeloDistributionSchedule.sol does not
+# implement the receive function nor does it allow ERC20 transfers. This is the only way I 
+# managed to give the CeloDistributionSchedule a balance.
 echo "Setting CeloDistributionSchedule balance..."
 cast rpc \
 anvil_setBalance \
 $CELO_DISTRIBUTION_SCHEDULE_ADDRESS $CELO_DISTRIBUTION_SCHEDULE_INITIAL_BALANCE \
 --rpc-url $ANVIL_RPC_URL
 
-# Fetch timestamp of the latest block
-LATEST_BLOCK_TIMESTAMP=$(
-    cast block \
-    latest \
-    --field timestamp \
-    --rpc-url $ANVIL_RPC_URL
-)
+# Run L2 migrations
+echo "Running L2 migration script... "
+forge script \
+$MIGRATION_L2_SCRIPT_PATH \
+--target-contract $MIGRATION_L2_TARGET_CONTRACT \
+--sender $FROM_ACCOUNT \
+--unlocked \
+$VERBOSITY_LEVEL \
+$BROADCAST \
+$SKIP_SIMULATION \
+$NON_INTERACTIVE \
+--rpc-url $ANVIL_RPC_URL || echo "L2 Migration script failed"
```

Comment:
Nit: add indentation to the lines that don't start a command

Reply:
That's done in https://github.com/celo-org/celo-monorepo/pull/11177/commits/066a6b2a0c4d0209f321158231e4a8fda39d3d20